addons/settings/XEH_preInit.sqf

```diff
@@ -0,0 +1,38 @@
+#include "script_component.hpp"
+
+// turn off RHS engine startup script
+RHS_ENGINE_STARTUP_OFF = 1; // any non-nil value turns this off
+
+if (isServer) then {
+    // set STUI settings for the BW experience (occlusion set mission side)
+    ["sthud_settings_squadbar", false, 0, "server"] call CBA_settings_fnc_set;
+    ["sthud_settings_removedeadviaproximity", true, 0, "server"] call CBA_settings_fnc_set;
```

Comment:
As far as I can tell this setting has no effect on anything

Reply:
Huh would you look at that... No only does it not do anything (from what I can see) the setting itself sets a slightly shorter GVAR (that also does nothing)....